ods_ci/tasks/Resources/Provisioning/Hive/provision.robot

```diff
@@ -32,7 +32,16 @@ Provision Cluster
     ${clustername_exists} =    Does ClusterName Exists
     ${template} =    Select Provisioner Template    ${provider_type}
     IF    ${clustername_exists}
-    ...    FAIL    Cluster name '${cluster_name}' already exists. Please choose a different name.
+        Log    Cluster name '${cluster_name}' already exists in Hive pool '${pool_name}' - Checking if it has a valid web-console      console=True
+        ${pool_namespace} =    Get Cluster Pool Namespace    ${pool_name}
+        ${result} =    Run Process 	oc -n ${pool_namespace} get cd ${pool_namespace} -o json | jq -r '.status.webConsoleURL' --exit-status    shell=yes
```

Comment:
Not sure whether check for webconsole valid url is good enough, but I don't have any better option myself anyway :upside_down_face: so hopefully it is :slightly_smiling_face:

Reply:
Up until now we just checked if the clusterclaim existed, but that is not indication if OCP install completed successfully. We had many times that Install completed, but the API and Web console were not created due to various errors (usually in PSI).

Reply:
Yeah, definitely better than checking the cluster claim.